docs/concepts/lite-agent.mdx

```diff
@@ -93,41 +93,46 @@ This example demonstrates the core features of a LiteAgent:
 For more complex scenarios, you can integrate LiteAgents into a Flow. Here's an example of a market research flow:
```

Comment:
what about renaming this concept also? Or will you keep LiteAgent in the documentation?